shared/my-sites/themes/themes-selection.jsx

```diff
@@ -46,12 +47,12 @@ const ThemesSelection = React.createClass( {
 	},
 
 	recordSearchResultsClick( theme, resultsRank ) {
-		const queryParams = ThemesListStore.getQueryParams();
+		const { queryParams, themesList } = this.props;
 		analytics.tracks.recordEvent( 'calypso_themeshowcase_theme_click', {
 			search_term: queryParams.search,
 			theme: theme.id,
 			results_rank: resultsRank + 1,
-			results: ThemesListStore.get(),
+			results: themesList,
```

Comment:
Noting here that `ThemesListStore.get()` should have actually read `ThemesListStore.getThemesList()` -- we had overlooked to change `get()` to `getThemesList()` when we started to use `createReducerStore()` to create our stores from reducers. That latter function however returns objects whose `.get()` method returns the entire store state. Thus, we were erroneously setting `results` to the contents of the entire `ThemesListStore` state instead (including query params and whatnot) instead of just the themes list. This has been fixed by this PR.